src/test/java/com/google/api/codegen/protoannotations/testdata/java_library.baseline

```diff
@@ -7653,55 +7659,7 @@ public class MyProtoClient implements BackgroundResource {
   private final MyProtoSettings settings;
   private final MyProtoStub stub;
 
-  private static final PathTemplate RETURN_PATH_TEMPLATE =
-      PathTemplate.createWithoutUrlEncoding("shelves/{shelf}/books/{book}/returns/{return}");
 
-  /**
-   * Formats a string containing the fully-qualified path to represent
-   * a return resource.
-   *
-   * @deprecated Use the {@link ReturnName} class instead.
-   */
-  @Deprecated
-  public static final String formatReturnName(String shelf, String book, String return_) {
```

Comment:
Is there any way to define a resource name outside of a message type?

Reply:
Hmm, doesn't look like the resource.proto gives us room for this, probably deliberately in order to enforce API guidelines.